ui/components/src/TimeSeriesTooltip/tooltip-model.ts

```diff
@@ -135,4 +135,16 @@ export const useMousePosition = (): CursorData['coords'] => {
 export type TooltipConfig = {
   wrapLabels: boolean;
   hidden?: boolean;
+  enablePinning?: boolean;
+  enableEChartsTooltip?: boolean;
```

Comment:
Does this prop differentiate whether to use the tooltip baked into echarts vs. the perses-based react tooltip or is it something a little different from that? If that is what it is, I wonder if this would be a little clearer (and enable supporting other tooltip kinds in the future) if it was an enum instead. Maybe something like `tooltipVariant?: 'echarts' | 'perses'` or something similar?  I'd also be cool with putting a typedoc comment above here clarifying the behavior, which is a lot less work.